src/main/resources/application.yaml

```diff
@@ -5,32 +5,31 @@ spring:
     password: null
     driverClassName: org.h2.Driver
     max-active: 15
-  profiles:
-    ### This is the FHIR version. Choose between, dstu2, dstu3, r4 or r5
-    active: r4
-  jpa:
-    properties:
-      hibernate.dialect: org.hibernate.dialect.H2Dialect
-      hibernate.search.model_mapping: ca.uhn.fhir.jpa.search.LuceneSearchMappingFactory
-      hibernate.format_sql: false
-      hibernate.show_sql: false
-      hibernate.hbm2ddl.auto: update
-      hibernate.jdbc.batch_size: 20
-      hibernate.cache.use_query_cache: false
-      hibernate.cache.use_second_level_cache: false
-      hibernate.cache.use_structured_entries: false
-      hibernate.cache.use_minimal_puts: false
-      hibernate.search.default.directory_provider: filesystem
-      hibernate.search.default.indexBase: target/lucenefiles
-      hibernate.search.lucene_version: LUCENE_CURRENT
+#  jpa:
+#    properties:
+#      hibernate.dialect: org.hibernate.dialect.h2dialect
+#      hibernate.search.model_mapping: ca.uhn.fhir.jpa.search.lucenesearchmappingfactory
+#      hibernate.format_sql: false
+#      hibernate.show_sql: false
+#      hibernate.hbm2ddl.auto: update
+#      hibernate.jdbc.batch_size: 20
+#      hibernate.cache.use_query_cache: false
+#      hibernate.cache.use_second_level_cache: false
+#      hibernate.cache.use_structured_entries: false
+#      hibernate.cache.use_minimal_puts: false
+#      hibernate.search.default.directory_provider: filesystem
+#      hibernate.search.default.indexbase: target/lucenefiles
+#      hibernate.search.lucene_version: lucene_current
 
   batch:
     job:
       enabled: false
 
 hapi:
   fhir:
-    defer_indexing_for_codesystems_of_size: 101
+    ### This is the FHIR version. Choose between, DSTU2, DSTU3, R4 or R5
+    fhir_version: DSTU3
```

Comment:
R4 should be default, as it's the current release

Reply:
now it is